metrics/grafana/ticdc.json

```diff
@@ -1,4 +1,46 @@
 {
+  "__inputs": [
+    {
+      "name": "DS_TEST-CLUSTER",
```

Comment:
```suggestion
      "name": "${DS_TEST-CLUSTER}",
```

Reply:
It is generated from Grafana.

Reply:
But tiup use `${DS_TEST-CLUSTER}` to do the replacement.
https://github.com/pingcap/tiup/blob/a33f0971969b5cce3d56784c760af3269368461d/templates/scripts/run_grafana.sh.tpl#L18

Reply:
never mind, I think this should work